### PR TITLE
Fix UI build process and upgrade Font Awesome

### PR DESCRIPTION
**Pull request by mraible**
- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/jhipster-registry/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-registry/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: src/main/webapp/manifest.webapp [hunk 1]
```diff
@@ -1,31 +1,31 @@
 {
-  "name": "JHipsterRegistry",
-  "short_name": "JHipsterRegistry",
+  "name": "Blog",
+  "short_name": "Blog",
```
Comment (ruddell) [inline src/main/webapp/manifest.webapp]: This line and the previous should say JHipsterRegistry

**Comment by mraible** [comment]
I'm not sure why this is failing with the following error:

```
ERROR in : Can't resolve all parameters for SessionStorageService in 
/Users/mraible/dev/jhipster/jhipster-registry/node_modules/ngx-webstorage/ngx-webstorage.d.ts: (?).
```

This error can be reproduced locally by running:

```
npm run webpack:prod
```

**Comment by mraible** [comment]
I tried to fix `npm test` too, but it fails with this strange error that I don't understand:

```
    Configuration error:
    
    Could not locate module ../../../main/webapp/app/shared/refresh/refresh.service mapped as:
    /Users/mraible/dev/jhipster/jhipster-registry/src/main/webapp/app/app/shared/refresh/refresh.service.
    
    Please check your configuration for these entries:
    {
      "moduleNameMapper": {
        "/app\/(.*)/": "/Users/mraible/dev/jhipster/jhipster-registry/src/main/webapp/app/$1"
      },
      "resolver": null
    }

```

**Comment by mraible** [comment]
Everything should be working as expected now.

**Comment by mraible** [comment]
It'd be nice if the Travis logs weren't so verbose from webpack. I added `JHI_DISABLE_WEBPACK_LOGS=true` to `.travis.yml`, but it doesn't seem to help. Any ideas @pascalgrimaud?

**Comment by murdos** [comment]
> It'd be nice if the Travis logs weren't so verbose from webpack. I added `JHI_DISABLE_WEBPACK_LOGS=true` to `.travis.yml`, but it doesn't seem to help. Any ideas @pascalgrimaud?

This variable is used in Webpack config in generator-jhipster: https://github.com/jhipster/generator-jhipster/search?q=JHI_DISABLE_WEBPACK_LOGS&unscoped_q=JHI_DISABLE_WEBPACK_LOGS but registry doesn't have the same webpack config.

**Comment by mraible** [comment]
@murdos I made the webpack changes to JHipster Registry, but the travis process builds with Docker. I made an attempt to solve it in https://github.com/jhipster/jhipster-registry/pull/355/commits/5ec8d34b80c011fcd6206d704fedda29223d29be.

**Pull request merged**